Src/GoogleApis.Auth.DotNet4/OAuth2/ServiceAccountCredential.cs

```diff
@@ -160,6 +178,39 @@ public ServiceAccountCredential(Initializer initializer) : base(initializer)
             return true;
         }
 
+        /// <summary>
+        /// Returns true if scopes are empty, which means the caller needs to 
+        /// invoke CreateScoped() to add scopes.
+        /// </summary>
+        public bool IsCreateScopedRequired
+        {
+            get
+            {
+                if (scopes == null || scopes.Count() == 0)

```

Comment:
IEnumberable.Count() is a method. 

https://msdn.microsoft.com/en-us/library/vstudio/bb338038(v=vs.110).aspx